straxen/rundb.py

```diff
@@ -40,6 +40,7 @@ def __init__(self,
                  local_only=False,
                  new_data_path=None,
                  reader_ini_name_is_mode=False,
+                 rucio_path = '/dali/lgrandi/rucio/',
```

Comment:
Not sure if we have anywhere in straxen dali related paths except for the contexts. Maybe you could change this into None and add this information simple to the context?

Reply:
There are some references to specific places/hosts even in this file:
https://github.com/XENONnT/straxen/blob/b176d08eec923876994f20fad328e714d070c22c/straxen/rundb.py#L28
https://github.com/XENONnT/straxen/blob/b176d08eec923876994f20fad328e714d070c22c/straxen/rundb.py#L91

But indeed perhaps it's nicer to put it in the context 1ac9c56